src/react/components/shell/loggedInRoute.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+import ApiService from "../../../services/apiService"
+
+export default function LoggedInRoute({component: Component, ...rest}) {
```

Comment:
as we called our component `SignInForm` and `SignInPage` maybe better name for the Route would be `SignedInRoute`?